microprofile/tests/tck/tck-fault-tolerance/pom.xml

```diff
@@ -32,6 +32,8 @@
     <properties>
         <!-- 3.0.0-JAKARTA -->
         <skipTests>true</skipTests>
```

Comment:
The TCK tests are still disabled. I was expecting this PR to re-enable them. Is there a dependency on another spec that must be resolved first?

Reply:
Enabled now